spec/spec_helper.rb

```diff
@@ -1,7 +1,7 @@
 require 'simplecov'
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
-SimpleCov.minimum_coverage 100
```

Comment:
I'm up for trying to get to 100%. Main worry is that the remaining 10% will just get bigger over time so it will just be more work upfront to regain 100% coverage.